app/views/hq/partners/_form.html.erb

```diff
@@ -2,32 +2,35 @@
   <%= render 'shared/errors', object: f.object %>
   <div class="form-group">
     <%= f.label :name %>
-    <%= f.text_field :name, class: "form-control" %>
+    <%= f.text_field :name, class: 'form-control' %>
   </div>
   <div class="form-group">
     <%= f.label :region %>
-    <%= f.text_field :region, class: "form-control" %>
+    <%= f.text_field :region, class: 'form-control' %>
   </div>
   <div class="form-group">
     <%= f.label :contact_details %>
-    <%= f.text_field :contact_details, class: "form-control" %>
+    <%= f.text_field :contact_details, class: 'form-control' %>
   </div>
   <div class="form-group">
     <%= f.label :province_id %>
     <%= f.collection_select :province_id,
-     @provinces, :id, :name , {},  class: "form-control" %>
+     @provinces, :id, :name , {},  class: 'form-control' %>
   </div>
   <div class="form-group">
     <%= f.label :status_id %>
     <%= f.collection_select :status_id,
-      @statuses, :id, :name, {}, class: "form-control" %>
+      @statuses, :id, :name, {}, class: 'form-control' %>
   </div>
   <div class="form-group">
     <%= f.label :start_date %>
-    <%= f.text_field :start_date, class: "form-control date-picker" %>
+    <%= f.text_field :start_date, class: 'form-control date-picker' %>
   </div>
   <div class="form-group">
-    <%= f.submit class: "btn btn-default" %>
+    <%= f.submit class: 'btn btn-primary' %>
+    <%= link_to('Cancel',
+                (@partner.id ? hq_partner_path(@partner.id) : hq_partners_path),
```

Comment:
In OSRA 333 for Users I implemented the selection of different cancel paths in a different way - basically by passing the cancel path to use in as a locals parameter to the form partial. I think it would make sense to adopt a common approach across the project. Which way do you guys prefer?